frontend/frontend-filters-links.php

```diff
@@ -507,7 +507,7 @@ protected function get_queried_term_id( $tax_query ) {
 		$taxonomy = $this->get_queried_taxonomy( $tax_query );
 
 		$field = $queried_terms[ $taxonomy ]['field'];
-		$term  = reset( $queried_terms[ $taxonomy ]['terms'] );
+		$term  = $queried_terms[ $taxonomy ]['terms'] ? reset( $queried_terms[ $taxonomy ]['terms'] ) : '';
```

Comment:
I think is sufficient to protect this `reset` call.
But what about `$queried_terms[ $taxonomy ]` $queried_terms is really null ? 
Is there no notice when we try to access it `$queried_terms[ $taxonomy ]['field']` or `$queried_terms[ $taxonomy ]['terms']`

What will happen behind with the get_terms() call where we use the `$field` and `$term` values ?

What about the `return reset( $terms )` at the end ? Because get_terms() can return another thing than an array.
https://developer.wordpress.org/reference/functions/get_terms/